.lingma-agents.yaml

```
@@ -0,0 +1,44 @@
+        - 14."检查是否存在低效算法或数据结构（如大数据量下使用嵌套循环）。"
+        - 15."确认是否存在重复计算情况（如在循环中执行相同的昂贵操作）。"
+        - 16."验证资源使用是否合理（数据库连接、线程等）。"
+        - 17."检查是否正确使用缓存提高性能。"
+        - 18."确认是否存在潜在的内存泄漏问题。"
+        - 19."确认是否对重要功能进行了单元测试验证。"
+        - 20."检查测试用例是否全面（正常路径、异常路径、边界条件）。"
+        - 21."确认是否使用Mock框架模拟外部依赖。"
+        - 22."验证测试是否遵循Arrange-Act-Assert模式。"
+        - 23."检查代码逻辑是否存在空指针风险。"
+    - path: "**/*.xml"
+      text: "1. 根据mybatis和mysql的用法检查sql中是否存在错误，例如类型错误、参数异常问题、表名异常问题、缺少</select>等闭合标签的问题"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)